pkg/agent/monitortool/latency_store.go

```diff
@@ -237,21 +237,38 @@ func (s *LatencyStore) DeleteStaleNodeIPs() {
 	}
 }
 
-// ConvertList converts the latency store to a list of TargetIPLatencyStats.
-func (l *LatencyStore) ConvertList() []stv1aplpha1.TargetIPLatencyStats {
+// ConvertList converts the latency store to a list of PeerNodeLatencyStats.
+func (l *LatencyStore) ConvertList(excludedNodeName string) []statsv1alpha1.PeerNodeLatencyStats {
 	l.mutex.RLock()
 	defer l.mutex.RUnlock()
 
-	entries := make([]stv1aplpha1.TargetIPLatencyStats, 0, len(l.nodeIPLatencyMap))
-	for nodeIP, entry := range l.nodeIPLatencyMap {
-		tempEntry := stv1aplpha1.TargetIPLatencyStats{
-			TargetIP:                   nodeIP,
-			LastSendTime:               metav1.NewTime(entry.LastSendTime),
-			LastRecvTime:               metav1.NewTime(entry.LastRecvTime),
-			LastMeasuredRTTNanoseconds: entry.LastMeasuredRTT.Nanoseconds(),
+	// PeerNodeLatencyStats should be a list of size N-1, where N is the number of Nodes in the cluster.
+	// We will have N-1 entries for each Node, each entry containing the latency stats for the connection,
```

Comment:
I'd suggest removing this comment line. It's not correct, and the other 2 lines are sufficient.

Reply:
Ok, that's right, I've updated it.